docs/ruleset_functions/get_SHW_types_and_SHW_use.md

```diff
@@ -0,0 +1,34 @@
+
+## get_SHW_types_and_SHW_use
```

Comment:
suggest get_SWH_uses_associated_with_SWH_building_area_types or something like that